apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -624,15 +628,16 @@ def grade(element_html, data):
     feedback_type = pl.get_string_attrib(element, "feedback", FEEDBACK_DEFAULT)
     answer_weight = pl.get_integer_attrib(element, "weight", WEIGHT_DEFAULT)
     partial_credit_type = pl.get_string_attrib(element, "partial-credit", "lcs")
+    allow_blank_submission = pl.get_boolean_attrib(element, "allow-blank", PL_ALLOW_EMPTY_SUBMISSION_DEFAULT)
 
     true_answer_list = data["correct_answers"][answer_name]
 
     final_score = 0
     feedback = ""
     first_wrong = -1
 
-    if len(student_answer) == 0:
-        data["format_errors"][answer_name] = "Your submitted answer was empty."
+    if not allow_blank_submission and len(student_answer) == 0:
+        data["format_errors"][answer_name] = "Your submitted answer was blank; you did not drag any answer blocks into the answer area."
```

Comment:
Shouldn't we just delete this code? I don't think the grade function will ever be run if there is already a format_error saved by the parse function. We can double check this with the other elements.

Reply:
What you said about the parse function makes sense (I'm not familiar with the exact behaviour if format_error is set tho 😓 ). I commented out the entire if check, and it doesn't seem to break anything... maybe someone else can chime in on this?

Reply:
I don't know if anyone will now the answer to this off the top of their head, the best thing to do is probably to just look at 4 or 5 of the other elements that implement this feature and see what they do. A lot of the other elements have much shorter code than this one, so it shouldn't take too long to do.

Reply:
I had a look at `pl-string-input`, `pl-number-input`, `pl-integer-input`, and it looks like you are correct; the other elements only set `format_error` in parse. I'll go ahead and remove the if block check then?

Reply:
Sounds good to me, thanks!